### PR TITLE
Selectors

### PR DESCRIPTION
**Pull request by OlgaBlud**
Hello! We've finished task #9 Selector. Please check our pull request.

### CODE REVIEW

File: client/components/selectors/selectors.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  </head>
+
+  <body>
+    <div class="selectors-container">
+      <div class="selector-group">
+        <div class="selector-block">
+          <label for="selector-block" class="selector-label input-label"
+            >Lable</label
+          >
+          <button class="dropdown-button"></button>
```
Comment (tbazko) [inline client/components/selectors/selectors.html:18]: this is usually called `trigger`. So, to keep consistent with your naming we can call this class `select-trigger`.

**Review by tbazko — CHANGES_REQUESTED** [review]
Please change format of `selector.js` to `selector.mjs`.

Overall great job! ❤️

File: client/components/selectors/selectors.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="selectors.css" />
+  </head>
+
+  <body>
+    <div class="selectors-container">
+      <div class="selector-group">
```
[the remark this answers is not in the document]
Comment (OlenaZH84) [reply client/components/selectors/selectors.html]: rename class as select-group

File: client/components/selectors/selectors.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        <div class="selector-block">
+          <label for="selector-block" class="selector-label input-label"
+            >Lable</label
```
[the remark this answers is not in the document]
Comment (OlenaZH84) [reply client/components/selectors/selectors.html]: fixed nitpick

File: client/components/selectors/selectors.html (hunk 1 above)
Comment (OlenaZH84) [reply client/components/selectors/selectors.html]: rename class as select-button

File: client/components/selectors/selectors.html [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+          <button class="dropdown-button"></button>
+          <input
+            type="text"
+            name="selected-reminder"
+            value=""
+            class="dropdown-input"
+            disabled
+          />
+        </div>
+        <ul class="dropdown-list">
```
[the remark this answers is not in the document]
Comment (OlenaZH84) [reply client/components/selectors/selectors.html]: rename class as select-options

File: client/components/selectors/selectors.html [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+          <li class="dropdown-list-item">After</li>
```
[the remark this answers is not in the document]
Comment (OlenaZH84) [reply client/components/selectors/selectors.html]: changed on select-option

File: client/components/selectors/selectors.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+  });
+  // Клик на Tab или Escape закрывает дропдаун
+  document.addEventListener("keydown", (e) => {
+    if (e.key === "Tab" || e.key === "Escape") {
+      dropdownList.classList.remove("dropdown-list-visible");
+      selectorBlock.classList.remove("selector-block-highlighted");
+      dropdownList.classList.remove("dropdown-list-highlighted");
+    }
+  });
+});
```
[the remark this answers is not in the document]
Comment (OlenaZH84) [reply client/components/selectors/selectors.js]: create function  closeSelect()

File: client/components/selectors/selectors.css [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+  display: none;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  border: 1px solid var(--border-unfocused);
+  border-radius: 8px;
+  width: 290px;
+  font-weight: 300;
+  font-size: 18px;
+  line-height: 28px;
```
[the remark this answers is not in the document]
Comment (OlenaZH84) [reply client/components/selectors/selectors.css]: used position: absolute

File: client/components/selectors/selectors.css [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+  height: 64px;
+  border: 1px solid var(--border-unfocused);
+  border-radius: 8px;
+}
+
+.selector-block-highlighted {
+  position: relative;
+  width: 290px;
+  height: 64px;
+  border: 3px solid var(--border-focused);
```
[the remark this answers is not in the document]
Comment (OlenaZH84) [reply client/components/selectors/selectors.css]: used box-sizing: border-box

**Review by artembykov — APPROVED** [review]

**Pull request merged**